client/lib/domains/index.js

```diff
@@ -158,6 +161,18 @@ function hasMappedDomain( domains ) {
 	return getMappedDomains( domains ).length > 0;
 }
 
+function isOpenSrsDomain( domain ) {
+	return domainEndsWithTld( domain, [ '.live' ] );
+}
+
+function isOpenHrsDomain( domain ) {
+	return domainEndsWithTld( domain, [ '.wales' ] );
+}
+
+function domainEndsWithTld( domain, tlds ) {
+	return some( tlds, ( tld ) => endsWith( domain, tld ) );
+}
```

Comment:
Too much abstraction? Hah! Let me tell you about my Java days... 😜

I agree that it might seem a bit over-engineered - but I did it for readability. I wanted to make it clear that the only difference between `isOpenSrsDomain` and `isOpenHrsDomain` is the array of domains they operate on. So I extracted the common logic (however little there is) to a function.

Your suggestion of extracting those arrays to constants is great and makes my intent here even clearer. I'll definitely do that.

Thanks for the hint about `partial` - it's a nice shortcut, although I must admit that I like the slightly more verbose arrow function approach here. You can almost read the line then - while using `partial` there introduces a weird, out-of-place word. Personal preference, I guess. I'll definitely keep `partial` in mind for future use 👍